src/wsock_message.erl

```diff
@@ -357,6 +357,8 @@ process_frame(open_close, _, Frame, Message) ->
   frame_to_complete_message(Frame, Message);
 process_frame(open_continue, _, Frame, Message) ->
   frame_for_fragmented_message(Frame, Message);
+process_frame(continue, begin_message, Frame, Message) ->
```

Comment:
@madtrick Hello. It has been a long time since I needed those changes also.
I know I've written a websocket client using wsock and igel and I know that I've made some changes so that I could authenticate using a http cookie.

Currently (9 months later) I don't remember why I needed to change the process_frame function.
It may have been a hack or it may have been a legitimate change. 
Is it possible from a wsock perspective that even the begining_message to be so fragmented that this change have a sense ?